cips/README.md

```diff
@@ -0,0 +1,69 @@
+# Celestia Improvement Proposal (CIP) process
+
+Read [CIP-1](./cip-1.md) for information on the CIP process.
+
+See the [WGs directory](./wgs/index.html) for Working Groups within the CIP process.
```

Comment:
[optional] we could move this link to the root level README

Reply:
good idea. on the site, do you think that it is clear enough from the sidebar, that the WG category is there?

Reply:
https://github.com/celestiaorg/CIPs/pull/179/commits/4c14da615fb8d76166236d564216f82c3f4934f6

it seems less out of place when it's in the root README, thank you